Hackney.Core/Hackney.Core.Sns/README.md

```diff
@@ -1,20 +1,39 @@
 # `Hackney.Core.Sns` NuGet Package
 
-#### Sns Gateway
+## Sns Gateway
 The `SnsGateway` implementation of the `ISnsGateway` interface allows the easy publishing of an event message to an Sns topic.
 The `ISnsGateway` interface is made available by using the `AddSnsGateway()` extension method during your application start-up.
 
-#### Shared Classes
+## Shared Classes
 - EntityEventSns - Model of the event message received by a function
 - EventData - Contains the data changed in an event
 - User - Contains information about a user triggering an event
 - EventTypes - Names all events we are currently using
 
-#### SnsIntialisationExtensions
+## SnsIntialisationExtensions
 
-The `SnsIntialisationExtensions` class configures an application to the Amazon Simple Notification Service (SNS). This will allow the application to interact with SNS Topics and subscriptions.
+The `SnsIntialisationExtensions` class configures an application to use the Amazon Simple Notification Service (SNS). This will allow the application to interact with SNS Topics and subscriptions.
 
-#### Usage
+## Usage
```

Comment:
I would leave this at level 4 - it is a sub-heading under SnsIntialisationExtensions